frontend/src/components/partners/messages.js

```diff
@@ -0,0 +1,135 @@
+  statistics: {
+    id: 'management.partners.stats.title',
+    defaultMessage: 'Statistics',
+  },
+  edit: {
+    id: 'management.partners.edit',
+    defaultMessage: 'Edit',
+  },
+  name: {
+    id: 'management.partners.fields.name',
+    defaultMessage: 'Name',
+  },
+  image: {
+    id: 'management.partners.fields.image',
+    defaultMessage: 'Image',
+  },
+  notAllowedPartners: {
+    id: 'management.messages.notAllowedPartners',
+    defaultMessage: 'You are not allowed to manage partners.',
+  },
+  partnerLink: { id: 'management.partners.link', defaultMessage: 'Link' },
+  startMapping: {
+    id: 'management.partners.currentProjects.startMapping',
+    defaultMessage: 'Start mapping',
+  },
+  partnerManage: {
+    id: 'management.partners.link.manage',
+    defaultMessage: 'Manage {entity}',
+  },
+  buildingsStats: {
+    id: 'management.partners.filter.buildingsStats',
+    defaultMessage: 'Building Edits',
+  },
+  contributors: {
+    id: 'management.partners.contributors',
+    defaultMessage: 'Contributors',
+  },
+  roadsStats: {
+    id: 'management.partners.filter.roadsStats',
+    defaultMessage: 'km of Roads',
+  },
+  editsStats: {
+    id: 'management.partners.filter.editsStats',
+    defaultMessage: 'Total Edits',
+  },
+  noPartnersGroup: {
+    id: 'partners.group.zero',
+    defaultMessage: 'No partners group yet',
+  },
+  edits: {
+    id: 'management.partners.activity.edits',
+    defaultMessage: 'Edits',
+  },
+  buildings: {
+    id: 'management.partners.activity.buildings',
+    defaultMessage: 'Buildings Added',
+  },
+  roads: {
+    id: 'management.partners.activity.roads',
+    defaultMessage: 'km of Roads Added',
+  },
+  users: {
+    id: 'management.partners.activity.users',
+    defaultMessage: 'Users',
+  },
+  readMore: {
+    id: 'partners.readMoreButton',
+    defaultMessage: 'Read more',
+  },
+  learnToMap: {
+    id: 'management.partners.learnToMap',
+    defaultMessage: 'Learn to Map',
+  }, 
+  myPartners: {
+    id: 'management.filter.buttons.myPartners',
+    defaultMessage: 'My Partners',
+  },
+  searchPartners: {
+    id: 'management.filter.textarea.searchPartners',
+    defaultMessage: 'Search partners...',
+  },
+  partner: {
+    id: 'management.partner',
+    defaultMessage: 'Partner',
+  },
+  partners: {
+    id: 'management.partners',
+    defaultMessage: 'Partners',
+  },
+  noPartnersFound: {
+    id: 'management.partners.list.empty',
+    defaultMessage: 'No partners were found.',
+  },
+  partnersInfo: {
+    id: 'management.partners.titles.partner_information',
+    defaultMessage: 'Partner information',
+  },
+  permalink:{
+    id: 'management.partners.titles.partner_permalink',
+    defaultMessage: 'Permalink',
+  },
+  primaryhashtag: {
+    id: 'management.partners.titles.partner_primaryhashtag',
+    defaultMessage: 'Primaryhashtag',
+  },
+  secondaryhashtag: {
+    id: 'management.partners.titles.partner_secondaryhashtag',
+    defaultMessage: 'Secondaryhashtag',
```

Comment:
Please change this to "Secondary hashtag(s)"

@CristianAmici @dgquail

Reply:
@emi420  If you start filling out the form, the cancel and save buttons appear, but only if you start completing the form, just like it works in the organizations section.

Reply:
I'm testing to confirm that everything works so I've tried filling the form, but the buttons doesn't appear:

<img width="551" alt="Screenshot 2024-06-12 at 17 43 06" src="https://github.com/hotosm/tasking-manager/assets/1226194/7fc9785b-838e-4377-9cfa-ad71213eab67">

Reply:
Oh I see, buttons are there but disabled, sorry I didn't see them at first!